metrics/web/test/auth/presentation/widgets/auth_form_test.dart

```diff
@@ -22,7 +22,13 @@ void main() {
         find.widgetWithText(MetricsTextFormField, AuthStrings.password);
     final submitButtonFinder =
         find.widgetWithText(MetricsPositiveButton, AuthStrings.signIn);
-
+    final suffixImageFinder = find.descendant(
+      of: find.byType(MetricsTextFormField),
```

Comment:
Should we ensure that the icon set as a suffix icon of the text field? This finder does not ensure that.